builds/snap/snapcraft.yaml

```diff
@@ -0,0 +1,139 @@
+name: easyrpg-player
+base: core20
+adopt-info: easyrpg-player
+summary: easyrpg-player
+description: |
+    EasyRPG Player is a program that allows to play games created with RPG Maker 2000 and 2003.
+    EasyRPG is developed by the amazing people running the [EasyRPG project](https://easyrpg.org/).
+
+    It aims to be a free (as in freedom) cross-platform RPG Maker 2000/2003 interpreter.
+    The main goal is to play all games created with them as the original game interpreter (RPG_RT) does.
+confinement: strict
+grade: stable
+
+apps:
+  easyrpg-player:
+    command: bin/desktop-launch $SNAP/usr/bin/easyrpg-player
+    plugs:
+      - x11
+      - wayland
+      - unity7
+      - opengl
+      - audio-playback
+      - joystick
+      - home
+
+    environment:
+      XDG_DATA_HOME: $SNAP_USER_COMMON/
+      DBUS_FATAL_WARNINGS: 0
+
+layout:
+  /etc/wildmidi:
+    bind: $SNAP/etc/wildmidi
+  /usr/share/midi/freepats:
+    bind: $SNAP/usr/share/midi/freepats
+
+parts:
+  liblcf:
+    plugin: autotools
+    source: https://github.com/EasyRPG/liblcf
+    source-type: git
+    build-environment:
+      - CXXFLAGS: "$CXXFLAGS -fuse-ld=gold -flto=$(nproc) -ffunction-sections -fdata-sections"
+      - LDFLAGS:  "$LDFLAGS  -fuse-ld=gold -flto=$(nproc) -Wl,--gc-sections"
+    autotools-configure-parameters:
+      - --prefix=/usr
+      - --disable-static
+    override-build: |
+      git fetch
+      snapcraftctl build
+      strip --strip-all $SNAPCRAFT_PART_INSTALL/usr/bin/lcf2xml
+      strip --strip-all $SNAPCRAFT_PART_INSTALL/usr/bin/lcfstrings
```

Comment:
Is it possible to launch the tools from the snap directly? Otherwise I would suggest to disable them alltogether.
(`--disable-tools` configure switch)

Reply:
Currently, the tools are not available - the only command that's exposed right now is the `easyrpg-player` command itself. I'll disable them for now and furthermore think about how to best integrate them at a later point in time.